src/features/ErrorPage/styled.js

```diff
@@ -0,0 +1,75 @@
+import styled from "styled-components";
+import { ReactComponent as ErrorIcon } from "./ErrorIcon.svg";
+
+const bpMobile = ({ theme }) => theme.breakpoint.mobile;
+
+export const StyledErrorPage = styled.div`
+    display: grid;
+    justify-items: center;
+    gap: 38px;
+    margin-top: 195px;
+
+    @media (max-width: ${bpMobile}px) {
+        gap: 14px;
+    }
+`;
+
+export const StyledErrorImage = styled(ErrorIcon)`
+    @media (max-width: ${bpMobile}px) {
+        width: 80px;
+    }
+`;
+
+export const Wrapper = styled.div`
+    display: grid;
+    justify-items: center;
+    gap: 24px;
+
+    @media(max-width: ${bpMobile}px) {
+        gap: 16px;
+    }
+`;
+
+export const ErrorText = styled.div`
```

Comment:
We'll have to think about headers hierarchy, but for now let's leave it that way